assembl/models/preferences.py

```diff
@@ -280,7 +280,8 @@ def validate_single_value(self, key, value, pref_data, data_type):
         elif data_type == "int":
             assert isinstance(value, int), "Not an integer"
         elif data_type == "json":
-            pass  # no check
+            # Will raise a JSONDecodeError if not a valid JSON
```

Comment:
Are you catching it somewhere ? What do you do then?

Reply:
No, let it raise an error if it fails, if it succeeds, then the validate function proceeds without an error.